src/msac.h

```diff
@@ -77,7 +77,7 @@ unsigned dav1d_msac_decode_bool_equi(MsacContext *s);
 unsigned dav1d_msac_decode_bool(MsacContext *s, unsigned f);
 unsigned dav1d_msac_decode_hi_tok(MsacContext *s, uint16_t *cdf);
 
-/* Supported n_symbols ranges: adapt4: 1-4, adapt8: 1-7, adapt16: 3-15 */
+/* Supported n_symbols ranges: adapt4: 1-3, adapt8: 1-7, adapt16: 3-15 */
```

Comment:
Ahh, this makes a lot of sense.  I was wondering why it was `1-4` when all of the others were `1-N - 1` for `adaptN`.